airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_manifest_only/pipeline.py

```diff
@@ -151,11 +168,18 @@ def _read_spec_from_file(self, spec_file: Path) -> dict:
     async def _run(self) -> StepResult:
         connector = self.context.connector
 
-        ## 1. Move manifest.yaml to the root level of the directory
+        ## 1a. Move manifest.yaml to the root level of the directory
         self.logger.info("Moving manifest to the root level of the directory")
         root_manifest_path = connector.code_directory / "manifest.yaml"
         connector.manifest_path.rename(root_manifest_path)
 
+        ## 1b. Move components.py to the root level of the directory if it exists
+        components_path = connector.python_source_dir_path / "components.py"
+        if components_path.exists():
```

Comment:
This moves the file itself, but it does NOT change the references to components in the manifest. Do they work as is and get resolved?